packages/astro/test/fixtures/content-layer/src/pages/missing.astro

```diff
@@ -1,7 +1,7 @@
 ---
 import { getEntry, render } from "astro:content"
 // Skipping the broken page in production so the build doesn't fail
-if(import.meta.env.PROD) {
+if(import.meta.env.PROD || import.meta.env.MODE === "production") {
```

Comment:
Maybe this change isn't required anymore if we change `isDev` to base on `runtimeMode` from `build()`

Reply:
`runtimeMode` is derived from `devOutput`, and `devOutput` controls this code that mutates `process.env.NODE_ENV` as either `development` or `production`.

Vite and Vite plugins rely on `process.env.NODE_ENV` to check if they're in production. e.g. `viteResolvedConfig.isProduction` is set based on `process.env.NODE_ENV === 'production'` (same for `import.meta.env.PROD`.

So in a way they're related, but `runtimeMode` itself is not accessible from Vite plugins as that's an Astro concept itself.

Reply:
The thing I care about is `env.command`, so I need a way to get a consistent value for that in both contexts. I've just pushed a change to use the actual command, instead of mode etc